charts/pega/templates/_pega-deployment.tpl

```diff
@@ -29,8 +29,12 @@ spec:
       labels:
         app: {{ .name }}
       annotations:
+{{- if .root.Values.annotations }}
```

Comment:
This should be relative to .node so the annotations block can be specified by tier.  

`{{- if .node.podAnnotations }}` seems more appropriate to me and aligns with the configuration pattern for annotations for ingresses and services.

Reply:
(And the `toYaml` below would need to be updated too)

Reply:
Done

Reply:
looks fine to me